lib/matplotlib/colorbar.py

```diff
@@ -267,7 +267,8 @@ def __call__(self):
         vmin = self._colorbar.norm.vmin
         vmax = self._colorbar.norm.vmax
         ticks = ticker.AutoMinorLocator.__call__(self)
-        return ticks[(ticks >= vmin) & (ticks <= vmax)]
+        rtol = (vmax - vmin) * 1e-10
+        return ticks[(ticks >= vmin - rtol) & (ticks <= vmax + rtol)]
```

Comment:
Just as a thought: Can it happen, that vmin > vmax? If so, neither the original nor the fixed code would work in that case.

Reply:
Yes but that gets checked and swapped inside the method before this code

Reply:
Confirmed that vmax and vmin get flipped if they are specified out of order.   Which maybe is bad behaviour?  But its what colorbar does...